x-pack/plugins/spaces/index.js

```diff
@@ -36,7 +37,9 @@ export const spaces = (kibana) => new kibana.Plugin({
     mappings,
     home: ['plugins/spaces/register_feature'],
     injectDefaultVars: function () {
-      return { };
+      return {
+        spaces: []
```

Comment:
Interesting... so if we don't define it here we can't provide it via the render call? That's super weird...

Reply:
Yeah - the Angular controller would actually fail because there isn't a "spacesProvider" registered with the system. At least that's what happened with my testing...I could be overlooking a better solution.

Reply:
I've seen crazier things, it works for me!